### ISSUE TITLE
Use ORCID IDs for translator credit and attribution

### ISSUE THREAD

**Issue by kcopas** [gbif/portal-feedback#2784]
It may require ORCID membership, but in principle we should be able to add translations as "works" to an authenticated user's ORCID profile. 

Translators are currently represented in a flat list as strings, not things: https://www.gbif.org/translators#list

To ensure that these volunteers receive credit and attribution for their work, it would be good to explore the changes needed to support a crediting system —ideally one that automates as much as sensible. 

Not sure where it starts. We have some translated digital documentation that use DOIs where ORCID IDs are present, which should move us into the CrossRef event realm (if and when it works). Should we consider adding a translator field to Contentful that prefers ORCIDs but accepts names,  which could then at minimum report back to ORCID?

However the details work out, I think this could be an interesting and sensible area for experimenting and exploring how to further extend benefits for our community by using ORCID (in a small test environment (n=~60)).

**Comment by dnoesgaard** [comment]
Just for experimenting, do you have an example of a GBIF document that has a DOI and has been translated by someone who has an ORCID?

**Comment by kcopas** [comment]
Both French and Spanish: https://doi.org/10.15468/doc-z79c-sa53

But I can see that we've got to add the translators to the colophon. I guess it would be nice to acknowledge contributors across all languages, regardless of what any one reader's viewing, no?

The missing credits thus far are:

Spanish: Anabela Plos (https://orcid.org/0000-0003-3877-7408), William Ulate (https://orcid.org/0000-0003-2863-2491), Miguel Vega, Cristina Villaverde (https://orcid.org/0000-0001-9244-399X)

French: André Heughebaert (https://orcid.org/0000-0002-7839-5300), Sophie Pamerlon (https://orcid.org/0000-0003-4736-7419), Pierre Radji (https://orcid.org/0000-0001-6597-8438), Brandy Parker, Carole Sinou (https://orcid.org/0000-0002-6718-6669), 

and, even though it's not yet published, Portuguese: Rui Figueira (https://orcid.org/0000-0002-8351-4028)

**Comment by MattBlissett** [comment]
We have the ability to add some text only to particular translations, if that's useful. ("This Spanish translation by ... corrections to ...")

**Comment by kcopas** [comment]
Yeah, agreed, understood—after thinking about it, though, my instinct is to credit translators in a block for all languages. Better to let everyone to know who's doing that work and not partition it off. Make sense?

Just means adding Translators / Traducteurs etc. as a standard block to include (or not).

**Comment by dnoesgaard** [comment]
We can add translators to DOI metadata as "other contributors" (no other type really fits), e.g.

for https://api.datacite.org/dois/10.15468/DOC-Z79C-SA53

(added Anabela and myself as a test)

```
"contributors": [
{
"name": "Plos, Anabela",
"givenName": "Anabela",
"familyName": "Plos",
"contributorType": "Other",
"nameIdentifiers": [
{
"schemeUri": "https://orcid.org",
"nameIdentifier": "https://orcid.org/0000-0003-3877-7408",
"nameIdentifierScheme": "ORCID"
}
],
"affiliation": []
},
{
"name": "Noesgaard, Daniel",
"givenName": "Daniel",
"familyName": "Noesgaard",
"contributorType": "Other",
"nameIdentifiers": [
{
"schemeUri": "https://orcid.org",
"nameIdentifier": "https://orcid.org/0000-0002-0407-1805",
"nameIdentifierScheme": "ORCID"
}
],
"affiliation": []
}
]
```
Now, the question is if "contributors" are included in the ORCID auto-update. I've posted a question to Datacite about this and I will also be able to check it in my own account (shortly)...

**Comment by dnoesgaard** [comment]
Ok, so that's a negative on "contributors" being added automatically: 

> Daniel,
> 
> we currently only look for creators. But we are happy to consider this as an enhancement, please suggest the idea at https://datacite.org/roadmap.html.
> 
> Best,
> 
> Martin
> 
> 
> Martin Fenner, MD
> DataCite Technical Director
> E: mfenner@datacite.org | ORCID

What do you think?

**Comment by kcopas** [comment]
Okay, maybe I misread the documentation and/or ontology. [This](https://members.orcid.org/api/resources/work-types) is what made me think it's possible.

Is Martin both DataCite and ORCID?

**Comment by dnoesgaard** [comment]
Martin's DataCite, but they're responsible for the automatic update of ORCID records to include contributions from Datacite.

We're restricted to the Datacite metadata scheme in this case. But I'll go ahead and put in a request to include contributors in the auto-update pipeline.
